lemur/plugins/lemur_aws/plugin.py

```diff
@@ -266,6 +225,49 @@ def clean(self, certificate, options, **kwargs):
         iam.delete_cert(certificate.name, account_number=account_number)
 
 
+class AWSDestinationPlugin(DestinationPlugin):
```

Comment:
Why copy and paste in this whole section from further up to down here? It makes it difficult to spot the differences

Reply:
good point, put there is not changes, except
sync_as_source_name = AWSSourcePlugin.slug

which is the reason why moving it down.